configure

```diff
@@ -232,6 +232,14 @@ case `$cc -v 2>&1` in
   *clang*) gcc=1 ;;
 esac
 
+if test $native -eq 0; then
+  avx2flag="-mavx2"
+  sse2flag="-msse2"
+  ssse3flag="-mssse3"
+  sse4flag="-msse4"
+  sse42flag="-msse4.2"
+  pclmulflag="-mpclmul"
+fi
```

Comment:
I would rather see this reversed.
So the flags are set appropriately initially, but if $native is enabled, only then zero the variables.
I think that would make more sense.

Reply:
Fixed.